homeassistant/components/sensor/github.py

```diff
@@ -0,0 +1,294 @@
+"""
+Support for GitHub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.github/
+"""
+from datetime import timedelta
+import logging
+import json
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN, CONF_USERNAME, CONF_PASSWORD, CONF_PATH, CONF_NAME,
+    CONF_SCAN_INTERVAL, ATTR_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['PyGithub==1.43.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_GROUP_AUTH = 'authentication'
+CONF_REPOS = 'repositories'
+CONF_SERVER_URL = 'server_url'
```

Comment:
I would like to suggest that we use `CONF_URL` (which is available in `const.py`) as other platforms do.

Reply:
Changed to `CONF_URL`. Also updated docs